runtime/gc_base/StackSlotValidator.hpp

```diff
@@ -60,7 +60,8 @@ class MM_StackSlotValidator : public MM_Validator {
 	 * @param message[in] a message to include with the report
 	 */
 	void reportStackSlot(MM_EnvironmentBase* env, const char* message);
```

Comment:
while here, fix '*' position for both params 
and also up for a few member fields